server/src/Services/ReleaseTestersService.cs

```diff
@@ -27,14 +28,45 @@ public Task<ReleaseTester> GetReleaseTesterById(int releaseTesterId)
         public Task<ReleaseTester> UpdateReleaseTester(int ReleaseTesterId, int State, string Comment)
         {
             ReleaseTester releaseTester = releaseTesters.UpdateReleaseTester(db, ReleaseTesterId, State, Comment);
+            if (GetReleaseState(releaseTester.ReleaseId)==1) 
+            {
+                Release release = releases.GetReleaseById(db, releaseTester.ReleaseId);
+                Project project = projects.GetProjectById(db, release.ProjectId);
+                var userIds = from userProject in userProjects.GetUsersForProject(db, project.Id) select userProject.UserId;
+                Email.sendEmail(users.GetUserEmailsByIds(db, userIds.ToList()), release.ReleaseName, project.Name, 2);
```

Comment:
Also what does 2 mean? Please don't use magic numbers. Create named readonly variable and the use it here.